modules/versor/0.0.1/patches/add_build_file.patch

```diff
@@ -0,0 +1,47 @@
+diff --git a/BUILD.bazel b/BUILD.bazel
+new file mode 100644
+index 0000000..23d04a9
+--- /dev/null
++++ b/BUILD.bazel
+@@ -0,0 +1,41 @@
++load("@bazel_skylib//rules:build_test.bzl", "build_test")
++load("@rules_license//rules:license.bzl", "license")
++
++package(
++    default_applicable_licenses = [":license"],
++    default_visibility = ["//visibility:public"],
++)
++
++license(
++    name = "license",
++    package_name = "versor",
++)
++
++licenses(["notice"])
++
++exports_files(["copyrights/COPYRIGHT"])
++
++cc_library(
++    name = "vsr",
++    srcs = glob(["src/space/*.cpp"]),
++    hdrs = glob([
++        "include/vsr/**/*.h",
++    ]),
++    copts = [
++        "-Wno-reorder",
++        "-Wno-unused-variable",
++    ],
++    features = [
++        "-parse_headers",
++    ],
++    includes = [
++        "include",
++        "include/vsr",
++        "include/vsr/detail",
++    ],
++)
++
++build_test(
++    name = "versor_build_test",
++    targets = [":vsr"],
++)
```

Comment:
I think we do not need to add a special test just for building. Building is tested via `presubmit.yml` already. We could probably get rid of the `bazel_skylib` dependency then as well.